yolox/evaluators/coco_evaluator.py

```diff
@@ -120,6 +121,7 @@ def evaluate(
         trt_file=None,
         decoder=None,
         test_size=None,
+        return_outputs=False
```

Comment:
The reason behind this is that when the evaluator finds the outputs for the validation samples, all the metrics are calculated and returned and the outputs are not. So in order to be able to log the visualizations on the dashboard, we need the output bounding boxes as well.
Hence, I added a `return_outputs` flag to the function to return outputs and the default value is False so that it doesn't break if the function is called anywhere else.